plugins/inputs/bigbluebutton/bigbluebutton.go

```diff
@@ -1,30 +1,51 @@
 package bigbluebutton
 
 import (
+	"crypto/sha1"
 	"encoding/xml"
 	"fmt"
+	"io/ioutil"
+	"net/http"
 
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/plugins/inputs"
 )
 
 type BigBlueButton struct {
-	URL         string `toml:"url"`
-	APIEndpoint string `toml:"api_endpoint"`
-	SecretKey   string `toml:"secret_key"`
+	URL              string `toml:"url"`
+	PathPrefix       string `toml:"path_prefix"`
+	SecretKey        string `toml:"secret_key"`
+	GetMeetingsURL   string
+	GetRecordingsURL string
```

Comment:
Please make those variables private by starting with lowercase letter as they should not be changed from outside AFAIK.

Reply:
Sorry. I'm new to Golang. This trick looks weird for me 😜.

Changed